packages/core/src/components/overlay/overlay.tsx

```diff
@@ -84,6 +84,11 @@ export interface IBackdropProps {
     backdropProps?: React.HTMLProps<HTMLDivElement>;
 
     /**
+     * Whether underlying content should be scrollable while the backdrop is visible.
+     */
+    backgroundScrollingDisabled?: boolean;
```

Comment:
Not off the top of my head, no. @giladgray said something in #183 about making this class prop-configurable. Not quite sure in what that meant - Gilad can you clarify? Happy to get rid of this if we want.